### ISSUE TITLE
[MPTV] Video player does not properly load (Possible solution inside)

### ISSUE THREAD

**Issue by CodeBrain-** [JetBoom/noxiousnet-issues#182]
The flash player in MPTV may or may not work for certain people. If it doesn't work then all you'll see is a black screen with the generic words for not having flash installed. I'm one of those people, but I have verified on Firefox, Chrome, IE, Edge and Steam Web browser that it is installed. Even the Cinema gamemode (non-Nox) works just fine. I posted about it on Facepunch after not getting a response on the forums and one developer found a possible solution:

thegrb93: "type="application/x-shockwave-flash" needs to be in both the object tag and the embed tag."

TL;DR: Player in MPTV is broken for some people, says they dont have flash despite them clearly having it. Possible solution is adding "type="application/x-shockwave-flash" to both the object and embed tags.
